apps/crn-server/src/migrations/1673349371-remove-working-groups-from-news.ts

```diff
@@ -0,0 +1,20 @@
+/* istanbul ignore file */
+import { Migration } from '@asap-hub/server-common';
+import { RestNews } from '@asap-hub/squidex';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class RemoveWorkingGroupsFromNews extends Migration {
+  up = async (): Promise<void> => {
+    await applyToAllItemsInCollection<RestNews>(
+      'news-and-events',
+      async (news, squidexClient) => {
+        if (news.data.type.iv === 'Working Groups') {
+          await squidexClient.delete(news.id);
+        }
+      },
+    );
+  };
+  down = async (): Promise<void> => {
+    // this is not needed
```

Comment:
I just added this comment because lint was giving an error that an empty function is not allowed

Anyway, I don't think this is needed right? If so, the idea is to move these records to a backup schema and move them back in case we want to rollback?

Reply:
nah too much effort, this is fine